### ISSUE TITLE
security vulnerability protobuf

### ISSUE THREAD

**Issue by renperez-cpi** [akeylesslabs/akeyless-js-cloud-id#17]
can someone please update protobuf to version 7.2.4. the current version has security vulnerability. > protobufjs@6.11.3 postinstall node_modules/akeyless-cloud-id/node_modules/protobufjs
> node scripts/postinstall

```
─────────────────────────────┼────────────────┼──────────┼───────────────────┼─────────────────────┼──────────────────────────────────────────────────────────────┤
│ protobufjs (package.json)   │ CVE-2023-36665 │ HIGH     │ 6.11.3            │ 7.2.4               │ prototype pollution using user-controlled protobuf message   │
│                             │                │          │                   │                     │ https://avd.aquasec.com/nvd/cve-2023-36665                   │
├─────────────────────────────┼────────────────┼──────────┼───────────────────┼─────────────────────┼──────────────────────────────────────────────────────────────┤
```

**Comment by renperez-cpi** [comment]
@orlangure do you know who can help with this?

**Comment by orlangure** [comment]
@AngelRefael

**Comment by renperez-cpi** [comment]
@AngelRefael any update boss?

**Comment by renperez-cpi** [comment]
@orlangure anyone else who can help with this? i think angel is busy

**Comment by orlangure** [comment]
Try [these people](https://github.com/akeylesslabs/akeyless-js-cloud-id/graphs/contributors) if they still work at Akeyless.

**Comment by renperez-cpi** [comment]
@renanaAkeyless can you help out?

**Comment by renanaAkeyless** [comment]
@renperez-cpi We will make sure to update it soon. Thank you for bringing this issue to our attention.

**Comment by renperez-cpi** [comment]
@renanaAkeyless any update?

**Comment by levpachmanov** [comment]
Hey @renperez-cpi ,
We're part of a startup called Seal Security that mitigates software vulnerabilities in older open source versions by backporting/creating standalone security patches - enabling more straightforward remediation in cases like this. We created an protobufjs 6.11.3-sp1 that's vulnerability-free. As with all of our patches, it's open-source and available for free.

If relevant, check out [our GitHub repo](https://github.com/seal-community/patches/tree/main/npm/protobufjs/6.11.3/sp1) if you wish to learn more, or start using our [app](https://app.sealsecurity.io/?ref=ghprotobufjs).

Please feel free to reach us at [info@seal.security](mailto:info@seal.security) if you have any requests/questions.

**Comment by renanaAkeyless** [comment]
@renperez-cpi fixed in 2.2.0
